src/main/java/ladder/domain/Persons.java

```diff
@@ -0,0 +1,71 @@
+package ladder.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Persons {
+
+    private final static String DELIMITER = ",";
+    private final static String DELIMITER_ERROR_MESSAGE = "구분자는 ,를 입력해주세요.";
+    private final static int MIN_PERSON_NUMBERS = 2;
+    private final static String MIN_PERSON_NUMBERS_ERROR_MESSAGE = "참여 인원은 2명이상이어야 합니다.";
+    private final List<Person> persons;
+
+    public Persons(String... input) {
+        this(Arrays.stream(input)
+                .map(Person::new)
+                .collect(Collectors.toList()));
+    }
+
+    public Persons(String input) {
+        this(Arrays.stream(parse(input))
```

Comment:
다른 부 생성자를 호출해보도록 변경해 볼 수 있을 것 같습니다.
```suggestion
        this(parse(input));
```